### PR TITLE
tips for coders

### PR DESCRIPTION
**Pull request by KHUSHI-jai-n**
Closes #[issue number]

## If an article related PR

- [x] I have checked that my suggestion follows the Contribution guide

- [x]  I have added which issue this PR closes

- [x] I have 0 warnings from Danger.js

- [x] If I have added images, they are in /files/ folder

## If other PR

- [x] I have added which issue this PR closes

Thanks for reading my pull request!
I have updated it, hope you like my article.

### CODE REVIEW

File: practices/tips_for_coders.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+slug: tips-for-coders
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:2]: see comment about title and change the slug too

File: practices/tips_for_coders.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+author_github_username: KHUSHI-jai-n
+---
+
+# Tips to know if you are a coder(or want to become one)
+
+**TL;DR**
+
+Everyone of us can code. We just need a little encouragement and confidence in ourselves.
+
+## What Are The Tips for coders?
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:34]: I am suggesting changing it to "Who is the coder" including the content of it.

File: practices/tips_for_coders.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+9. Consistency
+
+Nobody became a pro by putting in 20 hours in a day, but a lot of people did it by putting in 1 hour for 20 days. Your brain needs time to digest the information, you feed it.
+
+10. Open Source can help you level up
+
+Open Source is a great way to learn by building amazing Softwares and tools. But be prepared to be constantly learning as you would be required to be constantly adapting to new technologies. 
+
+## Why You Might Want The Tips For Coders
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:76]: I suggest changing the title and content to "Why You Might Become a Coder"

File: practices/tips_for_coders.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+A good coder is not made in a day, a week or a month.
+It takes much grinding and hardwork to polish your skills. These tips are just to give you a roadmap and help you out in your coding journey.
+
+## Problems The Tips For Coders Helps to Solve
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:81]: I suggest changing the title and content to "Problems The Coder Solves" or "Problems The Coder Has to Solve When Starting" (+ content from the first section).

File: practices/tips_for_coders.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+It helps beginners to start a smoother journey.
+It helps the experienced ones to become more efficient and have trust in themselves.
+
+These tips might seem very normal and basic but are missed out by most of us so consider these all just to remind you of the best practices you should follow.
+
+## How to Implement the Tips For Coders?
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:88]: I suggest changing the title and content to "How to Become a Coder" + tips

File: practices/tips_for_coders.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+Just follow them daily throughout your coding journey.
+Or feel free to come back and read them again to build more confidence.
+They could prove to be very motivating and guide you to the right path.
+
+## Common Pitfalls of the Tips For Coders
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:94]: I suggest changing the title and content to "Common Pitfalls of the Path to Become a Coder".

File: practices/tips_for_coders.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+**Making it seem too easy**
+This might make you look at things lightly which is not actually true. Coding requires consistent efforts and practice.
+
+**May only follow half the tips**
+There are so many tips and advices that you can feel overwhelmed. But do not worry - once you make youself comfortable with coding, you will realize that they are here to make your life easier, not more cumbersome.
+
+**These are not just all**
+These are only few of the many tips you can get. But I felt they were the most important so added these in this article.
+
+## Resources for the Tips For Coders
```
Comment (prokopsimek) [inline practices/tips_for_coders.md:105]: I suggest changing the title and content to "Resources to Become a Coder"

File: practices/tips_for_coders.md (hunk 5 above)
Comment (KHUSHI-jai-n) [reply practices/tips_for_coders.md:88]: Sure I will do that!

File: practices/tips_for_coders.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  - coding
+  - computer
+  - journey
+  - programming
+  - software
+  - hardwork
+author_github_username: KHUSHI-jai-n
+---
+
+# Tips to know if you are a coder(or want to become one)
```
[the remark this answers is not in the document]
Comment (KHUSHI-jai-n) [reply practices/tips_for_coders.md]: I have made the suggested edits and updated the pull request. Please do checkout!

File: practices/start_as_a_coder.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+**These are not just all**
+These are only few of the many tips you can get. But I felt they were the most important so added these in this article.
+
+## Resources to Become a Coder
+
+**Find more tips**
+
+- Medium: [To get a whole bunch of tips you can refer to this article](https://medium.com/@tapajyoti-bose/25-tips-i-wish-i-knew-before-i-started-to-code-2c6c1af20b71)
+- Analytics Insight: [Tips to make your programming joourney easier](https://www.analyticsinsight.net/top-10-secret-coding-tips-to-make-your-programming-journey-easier/)
```
Comment (prokopsimek) [inline practices/start_as_a_coder.md:116]: Please update the link according to the title of the external article for the proper attribution of the source.

**Review by prokopsimek — CHANGES_REQUESTED** [review]
@KHUSHI-jai-n Hi, your updates are much better, but you haven't changed the content in all sections according to the new headlines. Please take a look once more and follow my notes. Thank you!

File: practices/start_as_a_coder.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+  - coding
+  - computer
+  - journey
+  - programming
+  - software
+  - hardwork
+author_github_username: KHUSHI-jai-n
+---
+
+# Tips to know if you are a coder(or want to become one)
```
[the remark this answers is not in the document]
Comment (prokopsimek) [reply practices/start_as_a_coder.md:27]:
```suggestion
# Start as a coder
```

**Pull request merged**